packages/mongodb-rag-core/src/conversations/ConversationsService.ts

```diff
@@ -245,6 +248,11 @@ export interface ConversationConstants {
 export interface ConversationsService {
   conversationConstants: ConversationConstants;
 
+  /**
+    Initialize the conversations service.
+   */
+  init: () => Promise<void>;
```

Comment:
doesnt really matter but maybe make optional? 

```suggestion
  init?: () => Promise<void>;
```